src/main/java/ca/corefacility/bioinformatics/irida/service/impl/sample/MetadataTemplateServiceImpl.java

```diff
@@ -145,24 +143,32 @@ public MetadataTemplateField saveMetadataField(MetadataTemplateField field) {
 		return fieldRepository.findAllMetadataFieldsByLabelQuery(query);
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
 	@Override
 	@Transactional
 	@PreAuthorize("permitAll()")
-	public Map<MetadataTemplateField, MetadataEntry> getMetadataMap(Map<String, MetadataEntry> metadataMap) {
```

Comment:
Can we update this method to be something like `convertStringToMetadataMap` or something?

Reply:
Renamed in 
4c71547